api/src/identity-access-management/domain/models/OrganizationIdentity.js

```diff
@@ -0,0 +1,27 @@
+import { OrganizationLearnerNotBelongToOrganizationIdentityError } from '../errors.js';
+
+export class OrganizationIdentity {
+  /**
+   *
+   * @param {number} id
+   * @param {boolean} isScoGarIdentityProvider
+   * @param {OrganizationLearnerIdentity} organizationLearnerIdentity
+   */
+  constructor({ id, isScoGarIdentityProvider, organizationLearnerIdentities }) {
+    this.id = id;
+    this.isScoGarIdentityProvider = isScoGarIdentityProvider;
```

Comment:
En soi l'OrganizationIdentity n'est pas un ScoGarIdentityProvider, mais peut en avoir un, c'est pourquoi il me semble que l'utilisation de `has` plutôt que `is` serait plus adaptée : 

```suggestion
  /**
   * @param {number} id
   * @param {boolean} hasScoGarIdentityProvider
   * @param {OrganizationLearnerIdentity} organizationLearnerIdentity
   */
  constructor({ id, hasScoGarIdentityProvider, organizationLearnerIdentities }) {
    this.id = id;
    this.hasScoGarIdentityProvider = hasScoGarIdentityProvider;
```